### PR TITLE
Add nested HyperV storage perf test against Azure and HyperV 

### PR DESCRIPTION
**Pull request by xian123**

### CODE REVIEW

**Comment by iamshital** [comment]
@xian123 Thank you for your contribution. Before going through PR code, can I see your execution logs please? You can send it to v-shisav@microsoft.com.

File: XML/VMConfigurations/NestedVmTestsConfigurations.xml [hunk 1]
```diff
@@ -93,6 +105,18 @@
                     <Protocol>tcp</Protocol>
                     <LocalPort>22</LocalPort>
                     <PublicPort>22</PublicPort>
+                </EndPoints>
```
Comment (juhlee-microsoft) [inline XML/VMConfigurations/NestedVmTestsConfigurations.xml]: This line 108 is duplicated to the line 74. Why?

File: Libraries/HyperV.psm1 [hunk 2]
```diff
@@ -658,4 +677,13 @@ Function InjectHostnamesInHyperVVMs($allVMData)
             LogErr "Failed to inject $ErrorCount hostnames in HyperV VMs. Continuing the tests..."    
         }
     }
+}
+
+Function GetCred($user, $password)
```
Comment (juhlee-microsoft) [inline Libraries/HyperV.psm1]: Get-Cred()

File: XML/VMConfigurations/NestedVmTestsConfigurations.xml (hunk 1 above)
Comment (xian123) [reply XML/VMConfigurations/NestedVmTestsConfigurations.xml]: Because they're used for different setupType.

File: Libraries/HyperV.psm1 (hunk 2 above)
Comment (xian123) [reply Libraries/HyperV.psm1]: Changed the function name.

**Comment by juhlee-microsoft** [comment]
@xian123  please resolve the conflict for ready-to-merge.

**Comment by xian123** [comment]
Conflicts resolved @juhlee-microsoft Thanks for review.

**Comment by lizzha** [comment]
@xian123  the conflict is not resolved properly, you're including bunch of commits in this PR

**Comment by lizzha** [comment]
suggest to close this one, make a clean branch, and create a new PR

**Comment by xian123** [comment]
I follow @lizzha advice to close it and create a new PR based on the master.

**Pull request closed without merging**